### PR TITLE
Tweak the tentacle communications page to emphasise TLS not HTTPS

### PR DESCRIPTION
**Pull request by matt-richardson**
To help customers wanting to add http headers

[This customer](https://octopususergroup.slack.com/archives/C6UGLUWMQ/p1574438557358700) ran into some confusion with it.

### CODE REVIEW

File: docs/administration/security/octopus-tentacle-communication/index.md [hunk 1]
```diff
@@ -50,11 +50,17 @@ Tentacle plays the role of server and Octopus as the client:
...
 
-1. The Tentacle establishes the HTTPS connection with Octopus.
+1. The Tentacle establishes the TLS connection with Octopus.
 2. Octopus presents its certificate as the server certificate allowing the Tentacle to verify the identity of Octopus.
 3. The Tentacle presents its certificate as a client certificate so Octopus can verify the identity of the Tentacle.
 4. Once the identity of the Octopus and Tentacle have been established the connection is held open and Octopus will start issuing commands to the Tentacle.
 
+### Data transport
+
+Octopus uses [Halibut](https://github.com/OctopusDeploy/Halibut) to communicate. This is based on raw TCP, not HTTP - it is not possible to add HTTP headers to this communication channel.
```
Comment (jburger) [inline docs/administration/security/octopus-tentacle-communication/index.md]: [Nitpicking] I'd personally drop the word 'raw' as I don't think it really helps to clarify particularly for ESL readers.

```suggestion
Octopus uses [Halibut](https://github.com/OctopusDeploy/Halibut) to communicate. This is based on TCP, not HTTP - it is not possible to add HTTP headers to this communication channel.
```

File: docs/administration/security/octopus-tentacle-communication/index.md [hunk 2]
```diff
@@ -50,11 +50,17 @@ Tentacle plays the role of server and Octopus as the client:
...
+
+Both Tentacle and Server expose a simple page on the listening port to web browsers to allow you to confirm your configuration. Some security scanners detect this page and incorrectly assume that its a web server or a web app, and warn about self-signed certificates.
```
Comment (jburger) [inline docs/administration/security/octopus-tentacle-communication/index.md]: I think this is a good thing to call out :100:

**Comment by wordlee** [comment]
It's already merged, but fyi, I pushed a couple of minor line edits: https://github.com/OctopusDeploy/docs/commit/3b6959df73939777333f791767c8e17218644cbb

**Comment by matt-richardson** [comment]
Sorry, should've run it by you. Got over eager :)

**Pull request merged**